docs/en/framework/infrastructure/background-workers/tickerq.md

```diff
@@ -0,0 +1,141 @@
+# TickerQ Background Worker Manager
+
+[TickerQ](https://tickerq.net/) is a fast, reflection-free background task scheduler for .NET — built with source generators, EF Core integration, cron + time-based execution, and a real-time dashboard. You can integrate TickerQ with the ABP to use it instead of the [default background worker manager](../background-workers).
+
+## Installation
+
+It is suggested to use the [ABP CLI](../../../cli) to install this package.
+
+### Using the ABP CLI
+
+Open a command line window in the folder of the project (.csproj file) and type the following command:
+
+````bash
+abp add-package Volo.Abp.BackgroundWorkers.TickerQ
+````
+
+### Manual Installation
```

Comment:
No need to manual installation section, just link to the package detail page on abp.io, as done for the first package.